pkg/controller/daemon/daemon_controller.go

```diff
@@ -120,6 +121,11 @@ type DaemonSetsController struct {
 
 	// DaemonSet keys that need to be synced.
 	queue workqueue.RateLimitingInterface
+
+	// The DaemonSet that has suspended pods on nodes; the key is node name, the value
+	// is DaemonSet set that want to run pods but can't schedule in latest syncup cycle.
+	suspendedDaemonPods     map[string]sets.String
```

Comment:
why DaemonPods? The set contains the name of the daemon set not the pods name, no?